e2e-polybft/framework/test-cluster.go

```diff
@@ -771,8 +771,10 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 		require.NoError(t, err)
 
 		// add premine if token is non-mintable
-		err = bridge.mintNativeRootToken(addresses, tokenConfig, polybftConfig)
-		require.NoError(t, err)
+		if i == 0 {
```

Comment:
You need to include `premineNativeRootToken` and `finalizeGenesis` in this condition as well,